src/shave.cpp

```diff
@@ -354,9 +358,22 @@ void filterFeatures(vtzero::tile_builder* finalvt,
             return true; // skip to next feature
         }
 
-        // If evaluate() returns true, this feature includes properties that are relevant to the filter.
-        // So we add the feature to the final layer.
-        if (evaluate(mbgl_filter_obj, zoom, geometry_type, feature)) {
+        const mbgl::style::expression::EvaluationResult result = evaluate(mbgl_filter_obj, zoom, geometry_type, feature);
+        // expression hit error
+        if (!result) {
+            std::ostringstream msg;
+            msg << "[vtshaver] filter error: '" << result.error().message << "' skipping feature";
+            if (feature.has_id()) {
+              msg << " " << feature.id();
+            }
+            msg << " in " << std::string{layer.name()};
+            std::clog << msg.str() << "\n";
+            return true;
```

Comment:
This `return true` means that the feature will still be dropped (maintaining current behavior).

This new log will very useful for folks that use the `vtshaver` on the command line.

When vtshaver is deployed in an API-like environment this warning will not likely be useful, so I should likely add a way to turn it off as a future followup.